src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -1,33 +1,83 @@
+import { VisualizationType } from 'common/types/visualization-type';
 import { RequirementInstructions } from 'DetailsView/components/requirement-instructions';
+import * as styles from 'DetailsView/components/static-content-common.scss';
+import { createFastPassProviderWithFeatureFlags } from 'fast-pass/fast-pass-provider';
 import * as React from 'react';
+import * as Markup from '../../assessments/markup';
 
-export interface AdhocTabStopsTestViewProps {}
+export interface AdhocTabStopsTestViewProps {
+    configuration: VisualizationConfiguration;
+    featureFlagStoreData: FeatureFlagStoreData;
+    selectedTest: VisualizationType;
+}
 
-const howToTest: JSX.Element = (
-    <ol>
-        <li>
-            Locate the visual helper on the target page, it will highlight element in focus with an
-            empty circle.
-        </li>
-        <li>
-            Use your keyboard to move input focus through all the interactive elements in the page:
+export const AdhocTabStopsTestView = NamedFC<AdhocTabStopsTestViewProps>(
+    'AdhocTabStopsTestView',
+    props => {
+        const displayableData = props.configuration.displayableData;
+
+        const description = (
+            <p>
+                <Markup.Emphasis>
+                    Note: this test requires you to use a keyboard and to visually identify
+                    interactive elements.
+                </Markup.Emphasis>
+            </p>
+        );
+
+        const howToTest: JSX.Element = (
             <ol>
-                <li>Use Tab and Shift+Tab to navigate between standalone controls. </li>
                 <li>
-                    Use the arrow keys to navigate between the focusable elements within a composite
-                    control.
+                    Locate the visual helper on the target page, it will highlight element in focus
+                    with an empty circle.
+                </li>
+                <li>
+                    Use your keyboard to move input focus through all the interactive elements in
+                    the page:
+                    <ol>
+                        <li>
+                            Use <Markup.Term>Tab</Markup.Term> and{' '}
+                            <Markup.Term>Shift+Tab</Markup.Term> to navigate between standalone
+                            controls.{' '}
+                        </li>
+                        <li>
+                            Use the arrow keys to navigate between the focusable elements within a
+                            composite control.=
+                        </li>
+                    </ol>
                 </li>
             </ol>
-        </li>
-    </ol>
-);
-export const AdhocTabStopsTestView = NamedFC<AdhocTabStopsTestViewProps>(
-    'AdhocTabStopsTestView',
-    () => {
-        return <RequirementInstructions howToTest={howToTest} />;
+        );
+
+        const selectedTest = props.selectedTest;
+
+        const stepsText = (): string => {
```

Comment:
nit: this could probably be a bit cleaner; no need for steps text to be a function but something you can calculate in-line and then use.

Similarly, the given props object is not necessary, you can just refer to the variable you set as stepText and displayableData.title in the returned component itself.

Reply:
this is helpful, thank you!